### PR TITLE
SONAR-6651 Trim SCM logins

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

File: sonar-plugin-api/src/main/java/org/sonar/api/batch/scm/BlameLine.java [hunk 1]
```diff
@@ -68,8 +68,16 @@ public String author() {
...
+   * Sets author for this line.
+   * The string will be trimmed, and null will be set if it is empty.
+   */
   public BlameLine author(@Nullable String author) {
-    this.author = author;
+    String trimmedAuthor = (author == null) ? null : author.trim();
+    if(trimmedAuthor.isEmpty()) {
+      trimmedAuthor = null;
+    }
+    this.author = trimmedAuthor;
```
Comment (henryju) [inline sonar-plugin-api/src/main/java/org/sonar/api/batch/scm/BlameLine.java]: Replace by StringUtils.trimToNull(author) ?

Comment (dbmeneses) [inline sonar-plugin-api/src/main/java/org/sonar/api/batch/scm/BlameLine.java]: indeed

**Comment by henryju** [comment]
LGTM

**Comment by dbmeneses** [comment]
merged

**Pull request closed without merging**